### PR TITLE
Remove one bug in BatchNorm2d from GoogleNet

### PR DESCRIPTION
**Pull request by Anjiang-Wei**
This PR remove one redundant parameter in `BatchNorm2d`.
According to https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html#torch.nn.BatchNorm2d, the second parameter of `torch.nn.BatchNorm2d` corresponds to `eps=1e-05`, where the default value is preferred.

### CODE REVIEW

**Comment by dolong2110** [comment]
Hello how can I contribute to your repo?

**Comment by dolong2110** [comment]
Hello did you get my message, if you can see can I know the way to connect with you?

**Comment by dolong2110** [comment]
@weiaicunzai  how can I contribute to your repo?
